fenix/app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -127,6 +133,8 @@ class DefaultSessionControlControllerTest {
 
     @Before
     fun setup() {
+        mockkStatic(FragmentActivity::openToBrowserAndLoad)
```

Comment:
Opted to create and destroy the static mocks (over using scoped functions) in the set-up and tear down functions as `openToBrowserAndLoad` is used in numerous tests and also prevents vast code changes based on the formatting that would come with using ` mockkStatic(FragmentActivity::openToBrowserAndLoad) {`